crates/control/src/referee_pose_detection_filter.rs

```diff
@@ -85,6 +86,14 @@ impl RefereePoseDetectionFilter {
         let (is_referee_ready_pose_detected, did_detect_any_referee_this_cycle) =
             self.update(&context)?;
 
+        let is_standby = matches!(
+            context.game_controller_state.unwrap().game_state,
+            GameState::Standby
+        );
```

Comment:
This will panic when we do not have `game_controller_state`. Use this instead:
```suggestion
        let is_standby = matches!(
            context.game_controller_state,
            Some(GameControllerState {
                game_state: GameState::Standby,
                ..
            })
        );
```